### ISSUE TITLE
Error: async hook stack has become corrupted

### ISSUE THREAD

**Issue by eliw00d** [standard-things/esm#84]
I am not entirely sure if this is related to std/esm, but in the process of replacing require with import and module.exports with export in my project, I ran into this error:
```
$ node -r @std/esm index.js
Error: async hook stack has become corrupted (actual: 9, expected: 1)
 1: node::Environment::AsyncHooks::pop_ids(double) [/usr/local/bin/node]
 2: node::Start(v8::Isolate*, node::IsolateData*, int, char const* const*, int, char const* const*) [/usr/local/bin/node]
 3: node::Start(uv_loop_s*, int, char const* const*, int, char const* const*) [/usr/local/bin/node]
 4: node::Start(int, char**) [/usr/local/bin/node]
 5: start [/usr/local/bin/node]
```
Any thoughts on what could be going on? 

I have the following unlockables:
```
"@std/esm": {
    "cjs": true,
    "esm": "js"
  }
```
I am on version 0.7.1 currently and have Node 8.4.0.

**Comment by jdalton** [comment]
Hi @eliw00d!

It may be related to [this Node issue](https://github.com/nodejs/node/issues/13906). Marking the issue invalid just because it's not a clear `@std/esm` issue. Please feel free to ping back if you narrow it down to `@std/esm`.

**Comment by jdalton** [comment]
Found and patched the issue in #111.
